source/docs/fedora_atomic_bare_metal_installation.md

```diff
@@ -18,44 +22,27 @@ Once your system has completed booting, the boot screen is displayed:
-
-This menu allows you to configure your installation in the order you choose. Each menu item leads to an individual configuration screen. For description of these sub-screens, see the corresponding sections of the Fedora Installation guide:
+After media check, you are directed to the welcome screen.  This interactive install is exactly the same as installing standard Fedora Server, so you can use [Fedora's guide to the Anaconda GUI](https://docs.fedoraproject.org/en-US/Fedora/24/html/Installation_Guide/sect-installation-graphical-mode.html).
 
-- [*Time & Date*](http://docs.fedoraproject.org/en-US/Fedora/20/html/Installation_Guide/s1-timezone-x86.html) - lets you configure date, time, time zone, and NTP (Network Time Protocol).
-- [*Keyboard*](http://docs.fedoraproject.org/en-US/Fedora/20/html/Installation_Guide/sn-keyboard-x86.html) - here you can select keyboard layouts to use on your system.  
-- [*Language Support*](http://docs.fedoraproject.org/en-US/Fedora/20/html/Installation_Guide/language-support-x86.html) - this screen provides language options for your installation, it is identical to the welcome screen depicted above.
-- [*Installation Destination*](http://docs.fedoraproject.org/en-US/Fedora/20/html/Installation_Guide/s1-diskpartsetup-x86.html) - allows you to select storage devices and set partitioning. Note that it is recommended to use the default partitioning configured in the boot.iso image.
-- [*Network & Hostname*](http://docs.fedoraproject.org/en-US/Fedora/20/html/Installation_Guide/sn-Netconfig-x86.html) - provides a network management interface.
+The one thing which is different is disk partitioning.  Post-install, Atomic will automatically use some of the unused space on your primary LVM partition to create a Docker pool partition.  You will need this partition in order to have space for containers.  As such, we recommend leaving half your available disk space as unallocated LVM partition space.
 
 Once you configured all required settings, the orange warning signs disappear and you can start the installation by pressing *Begin Installation*.
 
-While the installation proceeds, you can specify the user settings:
-
-![Fedora Atomic Final Screen](final_screen1.png "Fedora Atomic Final Screen")
-
-[Select a root password](http://docs.fedoraproject.org/en-US/Fedora/20/html/Installation_Guide/s1-progresshub-x86.html#sn-account_configuration-x86) and [create a user](http://docs.fedoraproject.org/en-US/Fedora/20/html/Installation_Guide/sn-firstboot-systemuser.html) if desired. When the installation is finished, you are prompted to reboot the system by pressing *Reboot*.
+While the installation proceeds, you can specify user settings, such as a root password and adding one or more system users.  We strongly recommend that you do both, as Atomic Host has no GUI login and you will be unable to log in without one or more valid user logins. When the installation is finished, you are prompted to reboot the system by pressing *Reboot*.
```

Comment:
I just helped someone troubleshoot an issue with kubernetes that turned out to be a username collision with the `kube` system user.  Since anaconda doesn't check anywhere other than `/etc/passwd`, we probably want to minimally list the users *not* to create here.  

I don't know of any others than `kube` right now, nothing else in /usr/lib/passwd looks like something a user may pick on their own.